client/components/chip/index.tsx

```diff
@@ -0,0 +1,31 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+export type chipType = 'primary' | 'light' | 'warning' | 'alert';
+
+const Chip = ( props: {
+	message: string;
+	type?: chipType;
+	isCompact?: boolean;
+} ): JSX.Element => {
+	const { message, type = 'primary', isCompact } = props;
+
+	const classNames = [
+		'chip',
+		`chip-${ type }`,
```

Comment:
Does `type` set a default value of `'primary'` if none is passed in? Just checking because it looks like the old JS code had logic to set it to `primary` if not set (if I'm understanding it correctly!).

Reply:
yes, that's done during the destructuring 🙂 

See line 20, just 1 line above what's displayed in this thread 😅 

```js
const { /* ... */, type = 'primary', /* ... */ } = props;
```

Reply:
Ah I missed that 🤦 , thanks!